vulkaninfo/vulkaninfo.c

```diff
@@ -5407,7 +5384,11 @@ int main(int argc, char **argv) {
     free(gpus);
     free(objs);
 
-    AppDestroySurfaceChain(&inst);
+    for (int i = 0; i < inst.surface_ext_infos_count; ++i) {
```

Comment:
Once this loop is done, you would have to free `inst.surface_ext_infos` because it is heap memory.

Reply:
I am freeing it in Instance destructor.